### ISSUE TITLE
Display more context when replying to / following up a request

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/alaveteli#8289]
A user has got in touch with us here at WDTK to note that sometimes they want to review more context that just the previous message when drafting a reply, so it should be possible to review the entire correspondence chain.

---

I did think there was already a ticket for this, but I couldn't find it.

**Comment by garethrees** [comment]
Going to close in favour of https://github.com/mysociety/alaveteli/issues/4354 (which sort of covers this via the "reply inline" idea).
